packages/evolution-frontend/tests/ui-testing/testHelpers.ts

```diff
@@ -628,3 +689,44 @@ export const waitTextVisible = ({ context, text, isVisible = true }: { text: Pat
         }
     });
 };
+
+const continueOnInvalidPage: ContinueWithInvalidEntriesTest = async ({ context, text, currentPageUrl, nextPageUrl }) => {
+    const button = context.page.getByRole('button', { name: text });
+    await button.scrollIntoViewIfNeeded();
+    await button.click();
+    await expect(context.page.waitForURL(nextPageUrl, {timeout: 1000})).rejects.toThrow("Timeout")
+    await expect(context.page).toHaveURL(currentPageUrl);
+}
+
+/**
+ * Click the next page button when not all options are filled and check that we remain on the same page.
+ *
+ * @param {Object} options - The options for the test.
+ * @param {string} options.text - The text of the button to click.
+ * @param {string} options.currentPageUrl - The URL of the current page.
+ * @param {string} options.nextPageUrl - The URL of the page that the button would normally take us too.
+ */
+export const continueOnEmptyPage: ContinueWithInvalidEntriesTest = ({ context, text, currentPageUrl, nextPageUrl }) => {
```

Comment:
I don't think we need a different helper for empty or invalid pages. Both cases actually represent a pageWithInvalidWidgets, no matter the invalidity reason (empty mandatory or erroneous data)

Reply:
Not sure what you mean here. Is there already a test that does something similar? I don't seen any class named pageWithInvalidWidgets